spec.html

```diff
@@ -2593,183 +2593,120 @@ <h1>The Object Type</h1>
               <td>
-                The value retrieved by a get access of the property.
-              </td>
-            </tr>
-            <tr>
-              <td>
-                [[Writable]]
+                Any ECMAScript language type
               </td>
               <td>
-                Boolean
+                *undefined*
               </td>
               <td>
-                If *false*, attempts by ECMAScript code to change the property's [[Value]] attribute using [[Set]] will not succeed.
+                The value retrieved by a get access of the property.
               </td>
             </tr>
             <tr>
               <td>
-                [[Enumerable]]
-              </td>
-              <td>
-                Boolean
+                [[Writable]]
               </td>
               <td>
-                If *true*, the property will be enumerated by a for-in enumeration (see <emu-xref href="#sec-for-in-and-for-of-statements"></emu-xref>). Otherwise, the property is said to be non-enumerable.
+                data property
               </td>
-            </tr>
-            <tr>
               <td>
-                [[Configurable]]
+                Boolean
               </td>
               <td>
-                Boolean
+                *false*
               </td>
               <td>
-                If *false*, attempts to delete the property, change the property to be an accessor property, or change its attributes (other than [[Value]], or changing [[Writable]] to *false*) will fail.
+                If *false*, attempts by ECMAScript code to change the property's [[Value]] attribute using [[Set]] will not succeed.
               </td>
             </tr>
-          </table>
-        </emu-table>
-        <p>An accessor property associates a key value with the attributes listed in <emu-xref href="#table-accessor-property-attributes"></emu-xref>.</p>
-        <emu-table id="table-accessor-property-attributes" caption="Attributes of an Accessor Property" oldids="table-3">
-          <table>
-            <tr>
-              <th>
-                Attribute Name
-              </th>
-              <th>
-                Value Domain
-              </th>
-              <th>
-                Description
-              </th>
-            </tr>
             <tr>
               <td>
                 [[Get]]
               </td>
               <td>
-                Object | Undefined
+                accessor property
               </td>
               <td>
-                If the value is an Object it must be a function object. The function's [[Call]] internal method (<emu-xref href="#table-additional-essential-internal-methods-of-function-objects"></emu-xref>) is called with an empty arguments list to retrieve the property value each time a get access of the property is performed.
-              </td>
-            </tr>
-            <tr>
-              <td>
-                [[Set]]
+                Object | Undefined
```

Comment:
These will conflict with https://github.com/tc39/ecma262/pull/2602 depending on which order they land, FYI.

Reply:
Rebasing this didn't seem to show a conflict. What did you have in mind?

Reply:
done, as discussed